src/System.Runtime/tests/System/ArrayOfPointers.ArrayTests.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class ArrayTests
```

Comment:
I'm not familiar to what would it take for ILC to add support for pointer of arrays, but in the case the fix is not trivial I would like to unblock running the rest of the tests. If you think that separating to a different file is not a good plan for cases like this, then we could also use #ifdefs instead.